src/main/java/dan200/computercraft/core/lua/LuaJLuaMachine.java

```diff
@@ -412,6 +415,7 @@ public void execute()
                                             }
                                             catch( Throwable t )
                                             {
+                                                ComputerCraft.log.error( "Error running task", t);
```

Comment:
Suggestion: Make this extra debug log info a config option which defaults to off? Perhaps mention in the config that it will be useful to turn it on when debugging peripheral addons.